File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()
        new_s = ''.join([char for char in s if char.isalnum()])
        reversed_s = new_s[::-1]
        if(reversed_s == new_s):
            return True
        else:
            return False
        

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        int i=0;
        string newstr = "";
        while(s[i] != '\0'){
            if((s[i] >= 'a' && s[i] <= 'z') || (s[i] >= 'A' && s[i] <= 'Z') || (s[i] >= '0' && s[i] <= '9'))
                newstr += tolower(s[i]);
            i++;
        }
        int left = 0;
        int right = newstr.length() - 1;
        while(left<right){
            if(newstr[left] != newstr[right]){
                return false;
            }
            left++;
            right--;
        }
        return true;
    }
};